js/ElementRendererProvider.js

```diff
@@ -459,7 +485,7 @@ export default class ElementRendererProvider {
   }
 
   wrapAction(clickData: Object, preventDefault?: boolean, groupID?: String): Function {
-    return (event) => {
+    return (event, extElementPtr) => {
```

Comment:
What is `extElementPtr`? what is the different between `extElementPtr` and the event target?

Reply:
the `extElementPtr` was added because I needed a way to add reference to the associated `form` element from the element that is part of the associated `form` element. Beforehand, I am aware I utilized the [form attribute](https://caniuse.com/#feat=form-attribute). However, I realized that it is not supported on IE so therefore, I had to result to this solution. But if you know a better way, please let me know!

Reply:
I find it a bit confusing because  `extElementPtr` sounds the same as the `event.target`.
In addition - we already have a property to point the wrapping object - `groupID` can't we leverage that one for the same propose?

In the end this all affects the external API, and we should keep it as simple as we can.